collectors/plugins.d/README.md

```diff
@@ -32,17 +45,21 @@ Each of the external plugins is expected to run forever.
 Netdata will start it when it starts and stop it when it exits.
 
 If the external plugin exits or crashes, netdata will log an error.
-If the external plugin exits or crashes without pushing metrics to netdata,
-netdata will not start it again.
+If the external plugin exits or crashes without pushing metrics to netdata, netdata will not start it again.
+- Plugins that exit with any value other than zero, will be disabled. Plugins that exit with zero, will be restarted after some time.
+- Plugins may also be disabled by netdata if they are writing things that netdata does not understand.
 
 The `stdout` of external plugins is connected to netdata to receive metrics,
 with the API defined below.
 
 The `stderr` of external plugins is connected to netdata `error.log`.
 
+Plugins can create any number of charts with any number of dimensions each. Each chart can have its own characteristics independently of the others generated by the same plugin. For example, one chart may have an update frequency of 1 second, another may have 5 seconds and a third may have 10 seconds.
+
 ## Configuration
 
-This plugin is configured via `netdata.conf`, section `[plugins]`.
+Netdata will supply the environment variable `NETDATA_CONFIG_DIR` having the directory where configuration files are stored. It is up to the plugin to read the configuration it needs. `plugins.d` itself is configured via `netdata.conf`, section `[plugins]`. 
```

Comment:
`NETDATA_USER_CONFIG_DIR` (for user supplied) and `NETDATA_STOCK_CONFIG_DIR` (for netdata supplied) configuration files.

`NETDATA_CONFIG_DIR` still is supported but will be removed in a future release.

Reply:
> `plugins.d` itself is configured via `netdata.conf`, section `[plugins]`.

this is not accurate. At `netdata.conf` section `[plugins]` the `python.d.plugin` can be enabled or disabled.
At `netdata.conf` section `[plugin:python.d]` additional command line options can be given to it.
At `/etc/netdata/python.d.conf` the plugin has its own configuration file (usually for enabling/disabling its modules).
At `/etc/netdata/python.d/*.conf` is the configuration for each module.